docs/config/README.md

```diff
@@ -347,13 +347,14 @@ discharging_symbol = "💀"
 
 ### Battery Display
 
-The `display` configuration option is used to define when the battery indicator should be shown (threshold) and what it looks like (style).
+The `display` configuration option is used to define when the battery indicator should be shown (threshold) and what it looks like (style). The shown symbol can be specified as (symbol).
 If no `display` is provided. The default is as shown:
 
 ```toml
 [[battery.display]]
 threshold = 10
 style = "bold red"
+symbol = None
```

Comment:
Here is the thing, there is no such thing as `None` in TOML: https://github.com/toml-lang/toml/issues/146#issuecomment-25019344

```suggestion
```

Reply:
I removed the line.